manage.py

```diff
@@ -6,7 +6,7 @@
 from pygotham.factory import create_app
 from pygotham.manage import CreateAdmin, CreateEvent, CreateUser
 
-manager = Manager(create_app(__name__, ''))
+manager = Manager(create_app(__name__, ''), with_default_commands=False)
```

Comment:
This will remove both `runserver` and `shell`. Please add `shell` back, as it's a useful command to have. [This line](https://github.com/smurfix/flask-script/blob/02e1b3ee9ed9c0344153a0feb2719a05005b81d4/flask_script/__init__.py#L100) should do the trick.